src/app/organization/working-days/working-days.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="container">
+  <mat-card>
+
+    <div fxLayout="row" fxLayout.lt-md="column">
+      <h3 fxFlex="20%">Working Days</h3>
+      <div fxFlex="80%" fxLayout="column" class="input checkboxes" *ngFor="let day of weekDays; let i = index;">
+        <mat-checkbox [(ngModel)]="weekDays[i].checked">{{day.value}}</mat-checkbox>
```

Comment:
We are using `reactive forms` in the entire web app not `template-driven forms` (ngModel). Have a look at the other components for reference and update the code accordingly. You can learn more about reactive forms at https://angular.io/guide/reactive-forms

Also, let's keep the container width half like other components which have shorter forms for reference you can see, Accounting -> Closing Entries -> Create